docs/elements.md

```diff
@@ -1200,6 +1200,8 @@ def generate(data):
 | `show-header`     | boolean | true    | Show the header row of a DataFrame in default mode. (No effect in `text` mode.)                                                                             |
 | `show-index`      | boolean | true    | Show the index column of a DataFrame in default mode. (No effect in `text` mode.)                                                                           |
 | `show-dimensions` | boolean | true    | Show a footer with the dimensions of a DataFrame in default mode. (No effect in `text` mode.)                                                               |
+| `show-dtype`      | boolean | false   | Append a row to a DataFrame containing column data types for display. (No effect in `text` mode.)                                                           |
+| `add-line-breaks` | boolean | false   | Add line breaks for Python dictionaries or lists.                                                                                                           |
```

Comment:
Does this need to be configurable? In other words, is there any reason folks might *not* want to show line breaks?

Reply:
Yes. For short dictionaries like `{1:2, 2:4, 3:6}` it may be more convenient than having multiple lines. Also having the ability to default to false means this feature won't change the way people's existing questions look.